docs/getting-started/retrieving-keys.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: default
+---
+
+# Key Retrieval Guide
+
+This page explains how to retrieve keys from the server via a mobile application
+
+## Prerequisites
```

Comment:
This is pretty repetitive and could be out of sync easily.

Reply:
Moved to shared component